sdk/keyvault/azkeys/client.go

```diff
@@ -47,7 +47,7 @@ func (c *ClientOptions) toConnectionOptions() *policy.ClientOptions {
 }
 
 // NewClient returns a pointer to a Client object affinitized to a vaultUrl.
-func NewClient(vaultUrl string, credential azcore.TokenCredential, options *ClientOptions) (*Client, error) {
+func NewClient(vaultUrl string, credential azcore.TokenCredential, options *ClientOptions) (Client, error) {
```

Comment:
IIRC, wasn't the guideline to always return a client pointer?

Reply:
https://azure.github.io/azure-sdk/golang_introduction.html#golang-client-constructors